src/bundle/Resources/views/content/tabs/versions/table.html.twig

```diff
@@ -0,0 +1,32 @@
+<table class="ez-table-data">
+    <thead>
+        <tr>
+            <th>{{'locationview.versions.version'|trans({}, 'locationview')}}</th>
+            <th>{{'locationview.versions.modified.language'|trans({}, 'locationview')}}</th>
+            <th>{{'locationview.versions.author'|trans({}, 'locationview')}}</th>
+            <th>{{'locationview.versions.created'|trans({}, 'locationview')}}</th>
+            <th>{{'locationview.versions.last.saved'|trans({}, 'locationview')}}</th>
+        </tr>
+    </thead>
+    <tbody>
+    {% for version in versions %}
+        <tr>
+            <td>
+                {{ version.versionNo}}
+            </td>
+            <td>
+                {{ version.getContentInfo().mainLanguageCode }}
```

Comment:
I would write it `version.contentInfo.mainLanguageCode`, it feels more *template* :)